btrdbextras/dq.py

```diff
@@ -172,22 +191,20 @@ def __init__(self, streams):
             if not isinstance(stream, DQStream):
                 stream = DQStream(stream)
             dq_streams.append(stream)
-        
         # gets everything that a StreamSet has
         super().__init__(dq_streams)
-        # self._conn = streams[0]._btrdb
-        # self._distillates = self._get_distillates()
     
     @property
     def distillates(self):
         """
         Returns list of distillate streams
         """
```

Comment:
Do we want to have the option to access all distillates in a streamset? I'm not sure what the use case would be, and I think it could avoid errors to make users only access the distillates through the stream that they're tied to

Reply:
So you're suggesting getting rid of the `distillates()` method? I also can't really see the use case, so I'm fine getting rid of it. If we find that we need it later it'll be easy to add back in

Reply:
yeah I don't think this method is necessary for dqstreamset